charts/jaeger-operator/README.md

```diff
@@ -82,6 +81,13 @@ $ helm install jaegertracing/jaeger-operator --name my-release \
     --set rbac.create=false
 ```
 
+To install the chart without creating the CRDs (any files under `chart/crds`) make use of the `--skip-crds` flag. For example,
```

Comment:
Another thing that came to my mind. As `crds` folder is helmv3 feature only then we should add note to Prerequisites section. WDYT?

Reply:
Updated, I also clarified the cert-manager pre-requisite since ultimately the requirement is just for a certificate for the webhook.